src/update-manager.ts

```diff
@@ -106,30 +112,13 @@ export default class UpdateManager {
     }
     uri = termValue(uri)
 
-    // new logic for requests fron non-http resources
-    //   if not http(s) request
-    //     if has MachineEditableDocument triple -> EDIT AS LOCALFILE
-    //     else if !wac-allow write/append in header -> NOT EDITABLE
-    //     else
-    //       if accept-patch or ms-authorVia -> EDIT AS SPARQL
-    //       else -> EDIT AS LOCALFILE
-    if ((uri as string).slice(0, 4) != 'http') {
+    if ( this.isHttpUri(uri as string) ) {
```

Comment:
**issue**: You reverted the logic here from `!= http` to `isHttp`, I guess that is not intentional
**suggestion**: Unit Tests could catch such mistakes
**question:** Why do we need to cast `as string`?